rest-model/src/main/java/com/gentics/mesh/core/rest/node/NodeCreateRequest.java

```diff
@@ -42,6 +42,10 @@ public class NodeCreateRequest implements FieldContainer {
 	@JsonPropertyDescription("Whether the publish the node after creation.")
 	private boolean publish = false;
 
+	@JsonProperty(required = false)
+	@JsonPropertyDescription("Assign a requested language to the project, if none was so far.")
+	private boolean assignLanguage = false;
```

Comment:
The default value for this is not mentioned in the documentation and the default value is different from the default value of NodeUpdateRequest.
imho, both should be "true" (to reflect the "old" behaviour)